feed_flat_test.go

```diff
@@ -2,14 +2,19 @@ package getstream_test
 
 import (
 	"encoding/json"
+	"log"
 	"testing"
 	"time"
 
 	getstream "github.com/GetStream/stream-go"
 )
 
-func getFlatFeed(t *testing.T, client *getstream.Client) *getstream.FlatFeed {
-	feed, err := client.FlatFeed("flat", RandString(8))
+const (
+	flatFeedName = "flat"
+)
+
+func getFlatFeed(t *testing.T, client *getstream.Client, name string) *getstream.FlatFeed {
```

Comment:
Can you just make 2 functions, getFlatFeed and getRankedFeed. Then all the calls can just stay the same.